### PR TITLE
Progression 2 updates 2

### PR DESCRIPTION
**Pull request by hollasch**

### CODE REVIEW

File: images/img-2.23-book2-final.jpg
Comment (armansito) [inline images/img-2.23-book2-final.jpg:1]: Will you update the cover photo to match the new version? I see three nuances and just wanted to check that these are expected/intentional:

1. The center sphere has a new noise pattern.
2. The sphere on the bottom right is no longer fuzzy.
3. The Sahara and Arabian deserts on the texture mapped sphere seem to have these green blotches that aren't present in earthmap.jpg.

**Review by armansito — APPROVED** [review]
Updates to the text look good but I have some questions about the final render.

File: images/img-2.23-book2-final.jpg
Comment (hollasch) [reply images/img-2.23-book2-final.jpg:1]: 1. Expected.
2. Will investigate.
3. It does seem odd. Create an issue for this one and we'll put it on the backlog.

Comment (armansito) [reply images/img-2.23-book2-final.jpg:1]: I filed #1485

**Comment by hollasch** [comment]
I have created new PR #1491 without the image updates, which I'll do separately. I have merged this in, based on the approval of this PR.

**Pull request closed without merging**